README.md

```diff
@@ -359,6 +360,44 @@ This heading link of this heading is changed to following id
 - The `' '` (space) is replaced with `-` (dash), since `' '` is included in the first set
 - The '™' is replaced with _nothing_, since '™' is included in the second set
 
+#### URI encoding
+
+Non-ascii characters within link ids would be [URL encoded](https://en.wikipedia.org/wiki/Percent-encoding) default.
+
+```
+<!-- MarkdownTOC autolink="true" -->
+
+- [Ejemplos de español][ejemplos-de-espa%C3%B1ol]
+- [日本語の例][%E6%97%A5%E6%9C%AC%E8%AA%9E%E3%81%AE%E4%BE%8B]
+- [Примеры русского][%D0%9F%D1%80%D0%B8%D0%BC%D0%B5%D1%80%D1%8B-%D1%80%D1%83%D1%81%D1%81%D0%BA%D0%BE%D0%B3%D0%BE]
+- [中国的例子][%E4%B8%AD%E5%9B%BD%E7%9A%84%E4%BE%8B%E5%AD%90]
+
+<!-- /MarkdownTOC -->
+
+# Ejemplos de español
+# 日本語の例
+# Примеры русского
+# 中国的例子
+```
+
+But you can also disable this with `uri_encoding="false"` attribute.
```

Comment:
Rephrase: "As mentioned you can disable this by setting the `uri_encoding`attribute to `false`, like so: `uri_encoding="false"`.